### ISSUE TITLE
FEATURE: Loader position

### ISSUE THREAD

**Issue by MaeseppTarvo** [fxm90/GradientLoadingBar#2]
Hey, I think it would be awesome to be able to reposition the loader. I think it is achievable by making the setupConstraints(keyWindow: UIWindow){} function public, something like this:

```
public func setupConstraints(keyWindow: UIWindow, leadingAnchor: NSLayoutConstraint, trailingAnchor: NSLayoutConstraint, topAnchor: NSLayoutConstraint, heightAnchor: NSLayoutConstraint) {
        gradientView.leadingAnchor.constraint(equalTo: keyWindow.leadingAnchor).isActive = true
        gradientView.trailingAnchor.constraint(equalTo: keyWindow.trailingAnchor).isActive = true

        gradientView.topAnchor.constraint(equalTo: keyWindow.topAnchor).isActive = true
        gradientView.heightAnchor.constraint(equalToConstant: CGFloat(height)).isActive = true
    }
```

Another solution would be to add another variables in DefaultValues struct e.g public static `let leadingAnchor: NSLayoutConstraint!`

**Comment by fxm90** [comment]
In my opinion adding all the constraints to the initialiser would be too many arguments for a constructor, as this is not part of a normal configuration.

But I could make the method ```setupConstraints```public so that you can overwrite it within an subclass if help would help you?

**Comment by MaeseppTarvo** [comment]
@fxm90 I think that might be good also yea, but you can also make 2 separate Structs maybe? Like DefaultValues and CustomValues?

**Comment by fxm90** [comment]
So, I've just pushed a change where you can subclass ```GradientLoadingBar``` and overwrite ```setupConstraints()```. Unfortunately I didn't have time to write documentation at the moment, so I can't release a new version by now. 

But you can install it already via specifying the last commit in the podfile. For an example on how to subclass and customize check out the example project ;)

**Comment by fxm90** [comment]
Feature request is now available in release 1.1.6 🎉
